src/main.ts

```diff
@@ -54,7 +54,7 @@ app.config.globalProperties.$filters = {
     if (featureHierarchy) {
       const feature = featureHierarchy.find((featureItem: any) => featureItem.startsWith(featureKey))
       const featureSplit = feature ? feature.split('/') : [];
-      featureValue = featureSplit[2] ? featureSplit[2] : '';
+      featureValue = featureSplit[1] ? featureSplit[1] : '';
```

Comment:
Why we have changed the index from 2 to 1?

Reply:
Sir, actually I was extracting product features from productFeatures property and that has properties in a format like this.
![image](https://user-images.githubusercontent.com/52008359/149147963-b0f74814-e2c3-44f1-bbab-3660758090e8.png)
So that why I have changed this filter but I have changed it now and will extract it like other apps.

Reply:
Sir, I have already made changes to the code as per your recommendation.